dashboard/test/models/level_test.rb

```diff
@@ -263,4 +263,19 @@ def validate_karel_val(input, maze, initial_dirt, roundtrip=true)
     assert_nil level.embed
   end
 
+  test 'project template level' do
+    template_level = Blockly.create(name: 'project_template')
+    template_level.start_blocks = '<xml/>'
+    template_level.save!
+
+    assert !template_level.project_template_level
```

Comment:
Nice, did not know about assert_not. Might just do assert_nil since I do want nil not false